### DISCUSSION TITLE
Insecurly expl0it suddenly working and suddenly not working

### DISCUSSION THREAD

**Discussion by PenguinLord561** [3kh0/ext-remover#892] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I've been using insecurly for a while now and it suddenly stopped working one day after a restart.
I checked if chrome os updated, no. I've used caub so it should be locked to 119 until my school manually takes my chromebook and resets it.
For 2 days I just used LTMEAT flood to open pages that I needed but sometimes it crashes from the overload. One time when it crashed, the UI  went back to pre-119 (I could tell because it was a more boxy ui unlike 119 is more modern) and once I reset it again- it reverted. I checked insecurly just in case and it worked!! Any explamation for this?

**Comment by S-PScripts** [comment]
https://www.aboutchromebooks.com/news/what-happened-to-the-chromeos-119-release/
bit similar to v117
idk

**Reply by PenguinLord561** [reply]
@S-PScripts After having tabs open (for example playing back youtube) and doing LTMEAT Flood caused the Chromebook to crash and do weird things. The weird transition to a older ver was after I did flood multiple times. One other time it showed how chromeOS is repairing itself and today it logged me out of the school account, letting me login to my personal account (although it reverted back a few hours later)

**Comment by DLUNBOX** [comment]
#901 might work

**Reply by PenguinLord561** [reply]
@DLUNBOX unfortunately no, it shows 
![image](https://github.com/3kh0/ext-remover/assets/132022567/f05a31be-cbab-42ef-974f-fa5715670791)
i'm trying to reproduce the rollback again

**Reply by BasaltDelta** [reply]
it doesn't work for me either

**Reply by BasaltDelta** [reply]
I've been looking for ways but i haven't been able to find one.

**Comment by BasaltDelta** [comment, marked as answer]
maybe we can use proxies?

**Comment by BasaltDelta** [comment]
like proxy mirroring or smth?
